admin-base/ui.apps/src/main/js/apiImpl.js

```diff
@@ -495,14 +495,14 @@ class PerAdminImpl {
         })
   }
 
-  populateReferencedBy(path) {
-    return fetch('/admin/refBy.json' + path)
+  populateReferencedBy(path, sameTenant = false) {
+    return fetch(`/admin/refBy.json${path}${sameTenant ? '?sameTenant=true' : ''}`)
```

Comment:
using strings to build querys is a bit unintuitive but the fetch-api is not very advanced on that topic yet.
But you could use this approach: 
```js
fetch('https://example.com?' + new URLSearchParams({
    foo: 'value',
    bar: 2,
}))
```

https://stackoverflow.com/a/58437909